jobs/metricsgateway/spec

```diff
@@ -0,0 +1,133 @@
+    default: 5
+  autoscaler.metricsgateway.emitter.max_close_retry_count:
+    description: "The maximum retry times to close a websocket connection between emitter and metricsserver"
+    default: 5
+  autoscaler.metricsgateway.emitter.retry_delay:
+    description: "The duration between two websocket retrying"
+    default: 1s
+  autoscaler.metricsgateway.emitter.metricsserver_client.cert:
+    description: "PEM-encoded tls client certificate to connect to metricsserver"
+  autoscaler.metricsgateway.emitter.metricsserver_client.key:
+    description: "PEM-encoded tls client key to connect to metricsserver"
+  autoscaler.metricsgateway.emitter.metricsserver_client.ca_cert:
+    description: "PEM-encoded ca certificate of metricsserver"
+  autoscaler.metricsgateway.nozzle.rlp_addr:
+    description: "The address of ReverseLogProxy"
+  autoscaler.metricsgateway.nozzle.shard_id:
+    description: "The shard id for nozzle to fetch envelopes from ReverseLogProxy"
+    default: "CF_AUTOSCALER"
+  autoscaler.metricsgateway.nozzle.loggregator_rlp_tls.cert:
+    description: "PEM-encoded tls client certificate to connect to ReverseLogProxy"
+  autoscaler.metricsgateway.nozzle.loggregator_rlp_tls.key:
+    description: "PEM-encoded tls client key to connect to ReverseLogProxy"
+  autoscaler.metricsgateway.nozzle.loggregator_rlp_tls.ca_cert:
+    description: "PEM-encoded ca certificate of ReverseLogProxy"
+
+  autoscaler.policy_db.address:
+    description: "IP address on which the policydb server will listen"
+    default: "autoscalerpostgres.service.cf.internal"
+  autoscaler.policy_db.databases:
+    description: "The list of databases used in policydb database including name"
+  autoscaler.policy_db.db_scheme:
+    description: "Database scheme to be used to access policydb"
+  autoscaler.policy_db.port:
+    description: "Port on which the policydb server will listen"
+  autoscaler.policy_db.roles:
+    description: "The list of database roles used in policydb database including name/password"
+  autoscaler.policy_db.tls.ca:
+    default: ''
+    description: 'PEM-encoded ca certificate of tls database server'
+  autoscaler.policy_db.sslmode:
+    default: disable
+    description: "The sslmode to connect to postgres server"
+
+  autoscaler.policy_db_connection_config.max_open_connections:
+    default: 100
+  autoscaler.policy_db_connection_config.max_idle_connections:
+    default: 10
+  autoscaler.policy_db_connection_config.connection_max_lifetime:
+    default: 60s
+
+  autoscaler.metricsgateway.require_consul:
+    description: "Whether to start consul agent"
+    default: false
+  
+  autoscaler.metricsgateway.health.port:
+    description: "The listening port of health endpoint"
+    default: 6503
+
+  autoscaler.metricsgateway.hooks.pre_start:
+    description: "Script to run before starting metricsgateway"
+    default: ''
+    example: |
```

Comment:
again, why we need example?

Reply:
example are provided here to explain the required syntax.